examples/restaurant_visits/run_anonymize_values.py

```diff
@@ -0,0 +1,104 @@
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Demo of running PipelineDP locally, without any external data processing framework
+
+1. Install Python and run on the command line `pip install pipeline-dp absl-py`
+2. Run python run_without_frameworks.py --input_file=<path to data.txt from 3> --output_file=<...>
+"""
+
+from absl import app
+from absl import flags
+import pipeline_dp
+import pandas as pd
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string('input_file', 'restaurants_week_data.csv',
+                    'The file with the restaraunt visits data')
+flags.DEFINE_string('output_file', None, 'Output file')
+
+
+def write_to_file(col, filename):
+    with open(filename, 'w') as out:
+        out.write('\n'.join(map(str, col)))
+
+
+def main(unused_argv):
+    # Here, we use a local backend for computations. This does not depend on
+    # any pipeline framework and it is implemented in pure Python in
+    # PipelineDP. It keeps all data in memory and is not optimized for large data.
+    # For datasets smaller than ~tens of megabytes, local execution without any
+    # framework is faster than local mode with Beam or Spark.
+    backend = pipeline_dp.LocalBackend()
+
+    # Define the privacy budget available for our computation.
+    budget_accountant = pipeline_dp.NaiveBudgetAccountant(total_epsilon=1,
+                                                          total_delta=1e-6)
+
+    # Load and parse input data
+    df = pd.read_csv(FLAGS.input_file)
+    df.rename(inplace=True,
+              columns={
+                  'VisitorId': 'user_id',
+                  'Time entered': 'enter_time',
+                  'Time spent (minutes)': 'spent_minutes',
+                  'Money spent (euros)': 'spent_money',
+                  'Day': 'day'
+              })
+    df["count"] = 0
+    df_day_count = df[["day", "count"]].groupby("day").count()
+    day_count = [(row[0], row[1]["count"]) for row in df_day_count.iterrows()]
+    # day_count [(1, 300), ...]
+
+    # Create a DPEngine instance.
+    dp_engine = pipeline_dp.DPEngine(budget_accountant, backend)
+
+    params = pipeline_dp.AnonymizeValuesParams(
+        noise_kind=pipeline_dp.NoiseKind.GAUSSIAN,
+        # there are data for 7 days (partitions), so each visitor can contribute
+        # to not more than to 7 partitions
+        l0_sensitivity=7,
+        # Assume visitor can not go to more than 5 restaurants per day.
+        linf_sensitivity=5)
+
+    # Create the Explain Computation report object for passing it into
```

Comment:
This comment seems redundant. Maybe: "A report that helps to explain computations in the pipeline"?